### DISCUSSION TITLE
Offline Expedition 11 Hunter/Gatherer Milestone

### DISCUSSION THREAD

**Discussion by Shalmaneser1** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#42] [General]
Offline Expedition 11 Hunter/Gatherer Milestone
Unable to locate animal with 20.0 PAV. 
Has anyone completed this offline expedition? I have completed all other expedition 11 milestones.

**Comment by cwmonkey** [comment]
@Shalmaneser1 this comment may help you: https://github.com/leodium/leodium-ReLive-NoMansSky-Expeditions-Offline/discussions/39

**Reply by Shalmaneser1** [reply]
Thank you.
